package io.pivotal.errorHandling;

/**
 * Created by pivotal on 1/14/16.
 */
public class UserAlreadyExistsException extends Exception {
}
